WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -75,7 +75,7 @@
     <% end %>
 
     <%= f.input :name, disabled: !editable_fields.include?(:name) %>
-    <%= f.input :dob, as: :date_picker, disabled: !editable_fields.include?(:dob) %>
+    <%= f.input :dob, as: :date_picker, disabled: !editable_fields.include?(:dob), hint: "Enter your date of birth in the format YYYY-MM-DD" %>
```

Comment:
I'm not in love with copying this around, would it be possible to do this instead by changing en.yml as per http://stackoverflow.com/a/23649169/1739415? Maybe something like this:

```yml
en:
  simple_form:
    hints:
      defaults:
        dob: 'Enter your date of birth in the format YYYY-MM-DD'
```